plugins/copy-column/src/copy-column.ts

```diff
@@ -0,0 +1,175 @@
+import api, { Flatfile } from "@flatfile/api"
+import type { FlatfileEvent, FlatfileListener } from '@flatfile/listener'
+
+export type Primitive = string | number | null | boolean
+export type SimpleRecord = Record<string, Primitive>
+
+export function copyColumn() {
+    return (listener: FlatfileListener) => {
+      listener.on('workbook:created', async ({ context: { fileId, workbookId } }: FlatfileEvent) => {
+        const { data: sheets } = await api.sheets.list({ workbookId })
+
+        sheets.forEach(async (sheet) => {
+          const loadAllFields = await getAllFields(sheets, sheet.slug);
+          setSheetActions(sheet, [
+              copyColumnValuesBlueprint(loadAllFields),
+              ...(sheet.config.actions?.filter(
+              (a) =>
+                  a.operation !== 'copy-column'
+              ) || []),
+          ]);
+        })
+      })
+
+      listener.on("job:ready", { job: "sheet:copy-column" }, async (e) => {
+        const { records } = await e.data({ pageLength: 10_000 })
+        const { data: job } = await api.jobs.get(e.context.jobId)
+        const { key_copy_from, key_paste_to } = job.input
+        const patch = records.map(copyColumnValues(key_copy_from, key_paste_to))
+
+        await updateRecords(e.context.sheetId, patch)
+
+        await api.jobs.complete(e.context.jobId, {
+          outcome: {
+            message: `Data was successfully copied from ${key_copy_from} to ${key_paste_to}.`,
+          },
+        });
+      })
+    }
+}
+
+function copyColumnValues(from_key: string, to_key: string) {
+  return (record: Flatfile.Record_) => {
+    const obj = toSimpleRecord(record)
+    return {
+      id: record.id,
+      values: {
+        [to_key]: { value: obj[from_key] },
+      },
+    }
+  }
+}
+
+/**
+ * Updates records in a sheet. Bypasses API SDK in order to suppress hooks
+ */
+ async function updateRecords(sheetId: string, records: any): Promise<string> {
```

Comment:
I think you can replace this function with `updateRecords()` provided in the `@flatfile/util-common` package (check out the code [here](https://github.com/FlatFilers/flatfile-plugins/blob/main/utils/common/src/all.records.ts#L131)).